src/protocols/rdp/settings.c

```diff
@@ -1683,7 +1683,8 @@ void guac_rdp_push_settings(guac_client* client,
     if (guac_settings->remote_app != NULL) {
         freerdp_settings_set_bool(rdp_settings, FreeRDP_Workarea, TRUE);
         freerdp_settings_set_bool(rdp_settings, FreeRDP_RemoteApplicationMode, TRUE);
-        freerdp_settings_set_bool(rdp_settings, FreeRDP_RemoteAppLanguageBarSupported, TRUE);
+        freerdp_settings_set_bool(rdp_settings, FreeRDP_RemoteAppLanguageBarSupported, FALSE);
```

Comment:
Why is this changing to `FALSE`?

Reply:
@mike-jumper Any further thoughts on reverting this or leaving disabled?

Reply:
I think we should revert this unless things do not work without it disabled.

IIRC, the language bar is the UI component of Windows that allows switching between input methods, and would be needed for languages like Chinese and Japanese.